deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -32,6 +32,41 @@ spec:
         spec:
           description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
           properties:
+            inputs:
+              description: Inputs are named inputs of log messages
+              items:
+                description: InputSpec defines a source of log messages.
+                properties:
+                  app:
```

Comment:
@alanconway this is a nit more then anything, but should we expand this to 'application' to match the enum?

Reply:
Thanks Jeff, i missed this before doing the PR. 
I was facing issued with the generated code, so i was trying many things, including suspecting the name `application`. After several renamings, this got missed.
Updatig.